front/pages/no-workspace.tsx

```diff
@@ -41,8 +42,10 @@ async function fetchWorkspaceDetails(
 async function fetchRevokedWorkspace(
   user: UserTypeWithWorkspaces
 ): Promise<Workspace | null> {
-  const memberships = await Membership.findAll({
-    where: { userId: user.id },
+  // TODO(@fontanierh): this doesn't look very solid as it will start to behave
```

Comment:
This comment applies to the current logic not the new logic right?

Reply:
yep no change here, I'm planning to improve that.